server/components/auth/index.js

```diff
@@ -0,0 +1,45 @@
+/**
+ * Created by sourabh on 26/7/17.
+ */
+const jwt_token = require("jsonwebtoken");
+const compose = require("compose-middleware").compose;
+const verifyTsmsToken = (req, res, next) => {
```

Comment:
Add documentation for function:
/**
* <Purpose of function>
* @params - <parameters>
* @return - <return object>
*/